frameworks/kafka/universe/config.json

```diff
@@ -498,25 +535,31 @@
         },
         "queued_max_requests": {
           "title": "queued.max.requests",
-          "description": "The number of queued requests allowed before blocking the network threads ",
+          "description": "The number of queued requests allowed before blocking the network threads",
           "type": "integer",
           "default": 500
         },
+        "queued_max_request_bytes": {
+          "title": "queued.max.request.bytes",
+          "description": "The number of queued bytes allowed before no more requests are read",
+          "type": "integer",
+          "default": -1
+        },
         "quota_consumer_default": {
           "title": "quota.consumer.default",
-          "description": "Any consumer distinguished by clientId/consumer group will get throttled if it fetches more bytes than this value per-second",
+          "description": "DEPRECATED: Used only when dynamic default quotas are not configured for or in Zookeeper. Any consumer distinguished by clientId/consumer group will get throttled if it fetches more bytes than this value per-second",
```

Comment:
`[DEPRECATED]`?

Reply:
(I was wondering if there was some way to mark this in the schema directly.

Reply:
That's from the Kafka docs. They deprecated them.

I'll probably just remove the deprecated word.

Reply:
No, I like the deprecated. If this is automated, then let's leave it as is.